com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -19,6 +20,13 @@ public enum ShadowQuality
         SoftShadows,
     }
 
+    public enum SoftShadowQuality
+    {
+        Low,
+        Medium,
+        High,
+    }
+
```

Comment:
Remember to add docs for the enum + enum values

Reply:
done